package/bin/input_module_webex_api.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from webex_api_client import paging_get_request_to_webex, get_time_span
+from oauth_helper import get_valid_access_token
+
+from webex_constants import (
+   _WEBEX_FEDRAMP_BASE_URL
+)
+
+def collect_events(helper, ew):
+        
+        input_name = helper.get_input_stanza_names()
+        
+        # account args
+        opt_global_account = helper.get_arg("global_account")
+        account_name = opt_global_account.get("name")
+        client_id = opt_global_account.get("client_id")
+        client_secret = opt_global_account.get("client_secret")
+        stored_access_token = opt_global_account.get("access_token")
+        stored_refresh_token = opt_global_account.get("refresh_token")
+        base_endpoint = opt_global_account.get("endpoint")
+        
+        # input args
+        opt_start_time = helper.get_arg("start_time")
+        opt_end_time = helper.get_arg("end_time")
+        opt_webex_endpoint = helper.get_arg("webex_endpoint")
+        opt_webex_base_url = helper.get_arg("webex_base_url")
+        opt_query_params = helper.get_arg("query_params")
+        opt_is_gov_account = helper.get_arg("is_gov_account")
+        
+        start_time = None
+        end_time = None
+        
+        last_timestamp_checkpoint_key = f"{input_name}_webex_api_input_last_timestamp"
+        
+        # check if the start_time arg was provided
+        if opt_start_time:
```

Comment:
Since `start_time` is optional, what happens if the user doesn't provide a value for it? Should we skip the checkpointing logic in that scenario, or is there a default behavior we should handle?